lib/engine/game/g_1846/meta.rb

```diff
@@ -40,6 +40,13 @@ module Meta
             players: [2, 3, 4, 5],
           },
         ].freeze
+
+        def self.check_options(options, _min_players, _max_players)
+          optional_rules = (options || []).map(&:to_sym)
+          return unless (optional_rules & %i[first_ed second_ed_co]).length == 2
```

Comment:
This syntax was is due to rubocop not liking an if statement ( Style/GuardClause) and also not liking AND in a guard clause
(Style/UnlessLogicalOperators)